src/services/popover-orientation/popover-orientation.service.ts

```diff
@@ -1,8 +1,8 @@
-import { ElementRef, Injectable, OnDestroy, Renderer2 } from '@angular/core';
+import { ElementRef, Injectable, Renderer2 } from '@angular/core';
```

Comment:
Also I think Renderer2 is not being used for anything in this file.

Reply:
removed Renderer2